Mathlib/Control/Traversable/Equiv.lean

```diff
@@ -57,9 +56,8 @@ protected theorem comp_map {α β γ : Type u} (g : α → β) (h : β → γ) (
   simp [Equiv.map, Function.comp_def]
 
 protected theorem lawfulFunctor : @LawfulFunctor _ (Equiv.functor eqv) :=
-  -- Porting note: why is `_inst` required here?
```

Comment:
Why are you deleting this comment?

Reply:
`Equiv.functor` is not an instance, so it makes sense that it needs to be added to the context to be found. (Lean 3 trusts unification to find instances correctly, Lean 4 always looks for the instance and then checks that it is defeq to the instance found by unification, and that seems unlikely to change.)

Reply:
I won't insist on this, but a comment seems useful to me. Your call!